pkg/nas/nodeserver.go

```diff
@@ -352,19 +341,19 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 	}
 
 	// if volume set mountType as skipmount;
-	if opt.MountType == SkipMountType {
+	if opt.MountType == SkipMountType || runtimeVal == utils.RundRunTimeTag {
 		if features.FunctionalMutableFeatureGate.Enabled(features.RundCSIProtocol3) {
 			mountInfo := directvolume.MountInfo{
 				Source:     opt.Server,
 				DeviceType: directvolume.DeviceTypeNFS,
 				FSType:     "",
-				MountOpts:  strings.Split(opt.Options, ","),
+				MountOpts:  []string{opt.Options},
```

Comment:
We have already parsing the Options with `ParseMountFlags()`, just pass the slice over, don't join it?